cupyx/scipy/sparse/data.py

```diff
@@ -156,6 +158,20 @@ def _min_or_max(self, axis, out, min_or_max, sum_duplicates, non_zero):
 
         util.validateaxis(axis)
 
+        if axis is None:
+            if 0 in self.shape:
+                raise ValueError("zero-size array to reduction operation")
+
+            zero = self.dtype.type(0)
+            if self.nnz == 0:
+                return zero
+            if sum_duplicates:
+                self.sum_duplicates()
+            m = min_or_max(self.data.ravel())
+            if self.nnz != numpy.prod(self.shape):
+                m = min_or_max(zero, m)
```

Comment:
Could you investigate the `nonzero` option?

Reply:
`min_or_max(zero, m)` seems odd -- under the hood it's `cupy.min(zero, m)`, which doesn't make sense...

Reply:
I fixed to use `cupy.minimum` and `cupy.maximum`.